app/routes/signup.tsx

```diff
@@ -135,61 +142,198 @@ export const loader: LoaderFunction = async ({request}) => {
   })
 }
 
+const TEAM_MAP = {
+  BLUE: {
+    image: images.alexBlue,
+    label: 'Blue Team',
+    focusClassName: 'ring-team-blue',
+  },
+  RED: {
+    image: images.alexRed,
+    label: 'Red Team',
+    focusClassName: 'ring-team-red',
+  },
+  YELLOW: {
+    image: images.alexYellow,
+    label: 'Yellow Team',
+    focusClassName: 'ring-team-yellow',
+  },
+}
+
+interface TeamOptionProps {
+  team: keyof typeof TEAM_MAP
```

Comment:
With the change above, if this isn't happy, I think we could just do this:

```suggestion
  team: Team
```